### PR TITLE
fail the build if target is empty

### PR DESCRIPTION
**Pull request by OscarQQ**
Signed-off-by: OscarQQ <Yixin.Qian@ibm.com>

### CODE REVIEW

**Comment by OscarQQ** [comment]
Grinder/10559   target empty
Grinder/10560   target specified

**Comment by OscarQQ** [comment]
@llxia

**Comment by smlambert** [comment]
Thank you both for seeing this opportunity for improvement!  ;)

File: buildenv/jenkins/openjdk_tests [hunk 1]
```diff
@@ -115,6 +115,9 @@ def PLATFORM_MAP = [
     ]
 ]
 
+if ( params.TARGET == "" ) {
```
Comment (llxia) [inline buildenv/jenkins/openjdk_tests:118]: Any reason for not doing `if ( !params.TARGET ) {` ? By using it, we catch other invalid cases.

File: buildenv/jenkins/openjdk_tests [hunk 2]
```diff
@@ -115,6 +115,9 @@ def PLATFORM_MAP = [
...
+    assert false : "Please provide test target"
```
Comment (llxia) [inline buildenv/jenkins/openjdk_tests:119]: Please match the param name -> `Please provide TARGET value`

**Comment by OscarQQ** [comment]
updated, Grinder/10572 target empty, Grinder/10573 target provided

**Review by llxia — APPROVED** [review]
LGTM

**Pull request merged**